icons/package.svg

```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <line x1="16.5" y1="9.4" x2="7.5" y2="4.21" />
+  <line x1="16.5" y1="9.4" x2="7.6" y2="4.2" />
```

Comment:
Now it does still extend by a tiny bit. 😅 
```suggestion
  <line x1="16.5" y1="9.4" x2="7.55" y2="4.24" />
```

Reply:
Yeah, that’s why I made it less… 🙈